### ISSUE TITLE
Is the changelog.md file worth it?

### ISSUE THREAD

**Issue by conrad-vanl** [ApollosProject/apollos-prototype#144]
We started keeping up with a changelog.md file to document what's in releases. I'm starting to wonder if it's worth keeping up with. Instead, can we use commit messages in master as our changelog?

Since we do squash PRs, we end up with a single commit message every PR merge. If we make sure these commit messages are well written, our commit log becomes a much more interactive changelog. 

For releases, we can document changes within that release when we create the release. But I no longer believe the changelog file is worth it / adds any value.

Thoughts?

**Comment by IsaacHardy** [comment]
I do see redundancy in the change-log, however, I find the simplistic view to be easily understood when deciphering changes from each release. It is also a great resource for the Apollos Project board to understand if they were to explore the code.

**Comment by richarddubay** [comment]
I agree with Isaac about the board getting a high-level overview from the changelog. It would also be a great starting point when creating release notes for the various stores. But if there is another way to make all of that happen without having to keep a separate document ... that's something I'm game to try.

**Comment by burkeshartsis** [comment]
@conrad-vanl Since we all already talked about this can we close this issue?

**Comment by richarddubay** [comment]
I think the feeling that I'm getting is that people don't really want to bother with keeping up with a CHANGELOG. It's redundant with the commits that we push to `master`, we have to add to it manually, etc.

Part of me kind of feels like we think of the CHANGELOG kind of like we do documentation ... it's great to have, but no one really wants to create it or keep up with it. And we all have 100 things we have to do that we feel like are more important than dealing with those things. 

FWIW, I'm game for attempting something like the package that you mentioned @conrad-vanl. Once again referring to the past, for Holtzman we used to use `danger` and _require_ an updated CHANGELOG before the build would pass. That got to be too cumbersome because we ended up not wanting to update the CHANGELOG on every PR. 

Anyway, if there is anything that we could do to automate it's creation (if we deem it necessary) ... that would be awesome.
